openhtf/io/frontend/__main__.py

```diff
@@ -77,7 +77,8 @@
 
 
 class Station(mutablerecords.Record(  # pylint: disable=too-few-public-methods,no-init
```

Comment:
kind of a stretch to call this a Record now with all these extra methods, you might promote this to a real class, up to you, i don't feel too strongly about it, it's a small utility class either way.